relay-server/src/actors/project.rs

```diff
@@ -610,6 +616,7 @@ impl Project {
     ///
     /// The metrics will be keyed underneath this project key.
     pub fn insert_metrics(&mut self, metrics: Vec<Metric>) {
+        // TODO: rate limits
```

Comment:
@jan-auer I remember adding these TODOs together, but isn't this actually too late to rate limit incoming metrics? `InsertMetrics` is sent by the processor, don't we want to apply rate limits already on the fast path?